designs/administrator/olm/list-of-operators/list-of-operators.md

```diff
@@ -0,0 +1,76 @@
+---
+parent: Administrator
+version: 4.5
+---
+
+# List of Operators
+
+## All-projects view
+This is the Installed Operators page where “all-projects” has been selected from the project-selector. Info icons have been added next to the Operator Namespace and Managed Namespaces columns to provide the user more information about what these column titles mean.The user can sort the column via the sort icon (which looks like an arrow). This icon currently lives in the UI and is pre-existing functionality as part of the Namespace column.
+There are a few additions to the flow:
+
+
+1. We have introduced the Operator Namespace column. The Operator Namespace column lists the Namespace per Operator where the Operator is installed.
+
+2. In the Managed Namespaces column there are three different ways the Namespace information can appear.
+
+  a. “My-project” is representative of a single Namespace.
+
+   b. “All Namespaces” is a text only link that is associated with the Namespaces that fall across all Namespaces.
+
+   c. Thirdly, there is a multi-project view. For example, the Portworx Operator falls across multiple projects but not all projects, so in this example the Managed Namespaces column will have a popover on click. The Namespace names should be listed underneath as seen in the mockup.
+![all-projects-view](img/all-projects-view.jpg)
```

Comment:
Just wanted to mention it looks like the 'Installed Operators' list now has a 'Last Updated' column, maybe worth mentioning in the design?